src/symbolize.cc

```diff
@@ -837,18 +837,22 @@ static ATTRIBUTE_NOINLINE bool SymbolizeAndDemangle(void *pc, char *out,
 
 _END_GOOGLE_NAMESPACE_
 
-#elif defined(OS_WINDOWS)
+#elif defined(OS_WINDOWS) || defined(__CYGWIN__)
```

Comment:
I would suggest using `OS_CYGWIN` (from [here](https://github.com/google/glog/blob/5cfa9a28f51c41bcafd993f6e93a28daf977d276/src/utilities.h#L40)) for consistency.